packages/consoledot-api/src/transformers/topicTransformer.ts

```diff
@@ -7,6 +7,12 @@ import type {
 } from "ui-models/src/models/topic-config";
 import type { Bytes, Milliseconds } from "ui-models/src/types";
 
+export type UserEditable = {
+  "retention.ms": Milliseconds;
+  "retention.bytes": Bytes;
+  "cleanup.policy": "delete" | "compact" | "delete,compact";
+};
```

Comment:
Why this? The type is already defined in the ui-models. If the answer is "because we can't edit all those properties right now" then my suggestion is to pick the properties from that model, not duplicate them in a random place.